_data/simple-icons.json

```diff
@@ -4162,6 +4162,11 @@
             "hex": "E64415",
             "source": "https://www.spiegel.de"
         },
+        {
+            "title": "Detekt",
+            "hex": "45413C",
```

Comment:
As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#color-guidelines, we *usually* use https://meyerweb.com/eric/tools/color-blend/ for logos with gradients like this. I inputted the two start/stop colors from the gradient (https://meyerweb.com/eric/tools/color-blend/#7942EE:FF4651:1:hex) and got `#BC44A0` as the middle value. However, if you believe there is a good reason to pick one of the gradient colors over the other instead of the middle/mixed color, I'll hear you out :D

```suggestion
            "hex": "BC44A0",
```

Reply:
I felt that the reddish color was more meaningful given that detekt is a quality checking tool and reds are generally associated with the errors / complaints that the tool emits. The purplish midpoint seemed less semantically relevant and doesn't really connote anything about detekt in my mind.
Happy either way ultimately 🤷

Reply:
Actually, let's do neither the midpoint or red... let's do the purple!
![detekt dev_docs_gettingstarted_gradle](https://github.com/user-attachments/assets/d1d84f2d-2f8a-4fc5-afca-9154e4310d0d)

They seem to use it as their accent color on the docs, seems fitting. Does that work for you?

Reply:
Edited the suggestion above to use `#7942EE` 🫡

Reply:
@uncenter excellent! 🙏